servlet/src/main/java/io/undertow/servlet/UndertowServletMessages.java

```diff
@@ -232,4 +232,7 @@
 
     @Message(id = 10062, value = "No SecurityContext available")
     ServletException noSecurityContextAvailable();
+
+    @Message(id = 10063, value = "This Context has been already destroyed")
+    IllegalStateException nullDeploymentInfo();
```

Comment:
This is the wrong class to add the log message.
I know that having two classes for logs might be confusing, but this class contains errors messages that explain to user that something is missing or wrong on the config or code.
For this type of error, that is internal to the server (i.e., not caused directly by a problem in the deployment or deployment config), we will should use UndertowServletLogger instead.